specification/cognitiveservices/data-plane/Language/preview/2021-11-01-preview/analyzeconversations-authoring.json

```diff
@@ -0,0 +1,2703 @@
+        "name",
+        "language"
+      ],
+      "type": "object",
+      "properties": {
+        "type": {
+          "description": "Represents the project type.",
+          "enum": [
+            "conversation",
+            "workflow"
+          ],
+          "type": "string"
+        },
+        "settings": {
+          "description": "Gets or sets the project settings.",
+          "$ref": "#/definitions/ProjectSettingsObject"
+        },
+        "name": {
+          "description": "Gets or sets the new project name.",
+          "type": "string"
+        },
+        "multiLingual": {
+          "description": "Gets or sets a value indicating whether gets or sets whether the project is multilingual or not.",
+          "type": "boolean"
+        },
+        "description": {
+          "description": "Gets or sets the new project description.",
+          "type": "string"
+        },
+        "language": {
+          "description": "Gets or sets the new project culture.",
+          "type": "string"
+        }
+      }
+    },
+    "CluProjectGetObject": {
+      "required": [
+        "type",
+        "createdDateTime",
+        "lastModifiedDateTime",
+        "lastTrainedDateTime",
+        "lastDeployedDateTime",
+        "settings",
+        "name",
+        "language"
+      ],
+      "type": "object",
+      "properties": {
+        "type": {
+          "description": "Represents the project type.",
+          "enum": [
+            "conversation",
+            "workflow"
+          ],
+          "type": "string"
+        },
+        "createdDateTime": {
+          "format": "date-time",
+          "description": "Represents the project creation datetime.",
+          "type": "string"
+        },
+        "lastModifiedDateTime": {
+          "format": "date-time",
+          "description": "Represents the project creation datetime.",
+          "type": "string"
+        },
+        "lastTrainedDateTime": {
+          "format": "date-time",
+          "description": "Represents the project last trained datetime.",
+          "type": "string"
+        },
+        "lastDeployedDateTime": {
+          "format": "date-time",
+          "description": "Represents the project last deployed datetime.",
+          "type": "string"
+        },
+        "settings": {
+          "description": "Gets or sets the project settings.",
+          "$ref": "#/definitions/ProjectSettingsObject"
+        },
+        "name": {
+          "description": "Gets or sets the new project name.",
+          "type": "string"
+        },
+        "multiLingual": {
+          "description": "Gets or sets a value indicating whether gets or sets whether the project is multilingual or not.",
+          "type": "boolean"
+        },
+        "description": {
+          "description": "Gets or sets the new project description.",
+          "type": "string"
+        },
+        "language": {
+          "description": "Gets or sets the new project culture.",
```

Comment:
Is it a language e.g. "en" or a culture/locale e.g. "en-us", or either? Is this a BCP-47 code? Should be mentioned anywhere "language" or "locale" is.